src/js/clickable-component.js

```diff
@@ -196,9 +228,11 @@ class ClickableComponent extends Component {
   }
 
   /**
-   * Handle Blur - Remove keyboard triggers
+   * Called when a ClickableComponent component loses focus. This function turns of
+   * the watching for `keydown` events (so they will no longer call `this.handleKeyPress`
```

Comment:
Looks like "of" should be "off" but I feel like the terminology here should be "unbinds the listener for keydown events". Also missing a parentheses at the end.